src/Sarif.Converters/FortifyConverter.cs

```diff
@@ -147,8 +147,17 @@ public static Result ConvertFortifyIssueToSarifIssue(FortifyIssue fortify)
                     {
                         Locations = new List<AnnotatedCodeLocation>
                         {
-                            new AnnotatedCodeLocation { PhysicalLocation = source, Essential = true },
-                            new AnnotatedCodeLocation { PhysicalLocation = primaryOrSink, Essential = true }
+                            new AnnotatedCodeLocation
+                            {
+                                PhysicalLocation = source,
+                                Importance = AnnotatedCodeLocationImportance.Essential
+                            },
+
+                            new AnnotatedCodeLocation
+                            {
+                                PhysicalLocation = primaryOrSink,
+                                Importance = AnnotatedCodeLocationImportance.Essential
```

Comment:
Same here. If a converter actually doesn't distinguish between multiple importance levels, each step should be the default, imo.